datalad/distribution/tests/test_create.py

```diff
@@ -23,6 +23,7 @@
 from datalad.tests.utils import assert_not_in
 from datalad.tests.utils import assert_in
 from datalad.tests.utils import assert_raises
+from datalad.tests.utils import assert_equal
```

Comment:
Just because you do this on a regular basis: `assert_equal` isn't necessary, since we have the abbreviation `eq_` imported already. ;-)